test/IECoreMaya/FromMayaParticleConverterTest.py

```diff
@@ -96,6 +108,11 @@ def testEmitter( self ) :
 		self.assertEqual( particle["velocity"].data.getInterpretation(), IECore.GeometricData.Interpretation.Vector )
 		self.assert_( "mass" in particle )
 		self.assert_( particle["mass"].data.isInstanceOf( IECore.TypeId.FloatVectorData ) )
+		self.assert_( "radiusPP" in particle )
+		self.assert_( particle["radiusPP"].data.isInstanceOf( IECore.TypeId.FloatVectorData ) )
+		self.assert_( "userScalar1PP" in particle )
+		self.assert_( particle["userScalar1PP"].data.isInstanceOf( IECore.TypeId.FloatVectorData ) )
```

Comment:
I assume you meant to asset `userScalar2PP` is missing? And we should also test the other separators